bitbots_ball_filter/cfg/BallFilter.cfg

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+PACKAGE = "bitbots_ball_filter"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+import rospkg
+
+rospack = rospkg.RosPack()
+package_path = rospack.get_path('bitbots_ball_filter')
+
+gen = ParameterGenerator()
+
+group_ROS= gen.add_group("ROS", type="tab")
+group_filter= gen.add_group("Filter", type="tab")
+
+group_ROS.add("ball_subscribe_topic", str_t, 0, "", None)
+group_ROS.add("ball_position_publish_topic", str_t, 0, "", None)
+group_ROS.add("ball_movement_publish_topic", str_t, 0, "", None)
+group_ROS.add("ball_publish_topic", str_t, 0, "", None)
+group_ROS.add("filter_frame", str_t, 0, "", None)
```

Comment:
Don't dyn reconf the filter frame. Instead, set it in the launch file depending ob the environment variable

Reply:
which environment variable?